.github/workflows/ci.yml

```diff
@@ -0,0 +1,137 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - release/**
+
+  pull_request:
+
+jobs:
+  lints:
+    name: Lints
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install python dependencies
+        run: pip install --upgrade black flake8
+
+      - name: Run Black
+        run: black --check tests
```

Comment:
Q: should we use this directly instead of makefile for symbolic as well?

Reply:
Since we're going with explicit commands now, I would say yes. The makefile becomes an interface for the developer, and CI has it's own (equivalent or equal) commands.